tutorials/hello/hello_client/src/app/modules/hello/events/new_hello.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { BaseEvent, EventQueuer } from 'lisk-sdk';
+
+export const newHelloEventSchema = {
+	$id: '/hello/events/new_hello',
+	type: 'object',
+	required: ['senderAddress', 'message'],
+	properties: {
+		senderAddress: {
+			dataType: 'bytes',
+			fieldNumber: 1,
```

Comment:
format: 'lisk32'

Reply:
in that case, should `NewHelloEventData.senderAddress` have type string instead of Buffer?

Reply:
FYI I did not make the suggested change, because it was creating an error in the app. If you want to change the existing working code, please also provide a reasoning, what is the benefit of doing so.

Reply:
ah `format: 'lisk32'` was additional property you can set. it will validate the address format and with fromJSON/toJSON, it will automatically convert to the lisk32 format